packages/sui-decorators/test/server/error.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-unused-expressions */
+import {expect} from 'chai'
+
+import {error} from '../../src'
+
+describe.only('Error', () => {
+  it('Should return an array [null, resp] when the promise es resolved', async () => {
+    class Buzz {
+      @error()
```

Comment:
I know what it does because we talked about this but for me, the name does not give any clue about what it does.
Could we think about one more accurate? `responseWithErrorDTO` , `mappedWithError`, `errorIncluded` are proposals but, I don't like them so much neither. @carlosvillu

Reply:
Maybe `@inlineError()`

Reply:
Sounds better than my proposals. Anyway it still doesn't give information what it does.. Let's see if anyone can give us some light. 

If noone can't, definitely `@inlineError()` is the best.

Reply:
`@inlineError` sounds better than error. It could be something like `@errorResponseWrapper` too